app/views/about/help.md

```diff
@@ -4,6 +4,8 @@
 
 ## Main
 
+<div class="toc"></div>
+
```

Comment:
@afred: Not excited about this approach, but I could imagine them wanting some words above the ToC, and if we went the other way, something outside the MD would need to remember that this particular page is special, and I'd like to avoid that.

Open to suggestions.